core/controllers/profile.py

```diff
@@ -65,9 +65,7 @@ def get(self, username):
 
 class ProfileHandler(base.BaseHandler):
     """Provides data for the profile page."""
-
     PAGE_NAME_FOR_CSRF = 'profile'
```

Comment:
For consistency with other controllers in this file, please leave this as it was (with newlines surrounding it).